### ISSUE TITLE
Images get squished when Editor is small

### ISSUE THREAD

**Issue by blumelume** [notaapp/talk#393]
When I resize the editor window to small sizes or when I open the sidebar in a reasonably small window, the text-width gets adjusted correctly but images get squished.

Tested on: macOS 13.2.1 (Ventura) & High Sierra

window-width ca. 721px 

<img width="721" alt="Screenshot 2023-04-24 at 14 33 01" src="https://user-images.githubusercontent.com/30017507/233998789-e7d949a1-e355-4185-9f5a-6efb05a6f4c1.png">

window-width: ca. 1080px

<img width="1080" alt="Screenshot 2023-04-24 at 14 38 37" src="https://user-images.githubusercontent.com/30017507/233998799-a49c7cac-24a9-4055-b200-cb8210aacf29.png">

**Comment by astoilkov** [comment]
I will take a look at this soon and hopefully fix it. Thanks!

**Comment by astoilkov** [comment]
This is now fixed. The next release will include the fix.

**Comment by mathewmitchell** [comment]
Any estimate on when next release is coming?

**Comment by astoilkov** [comment]
Next month, probably. Is this bug a big issue for you?

**Comment by mathewmitchell** [comment]
No, not a big issues. Mainly asking out of curiousity since I vaguely remember reading that a new release would be forthcoming in late May.
